src/main/java/seedu/address/model/transaction/Date.java

```diff
@@ -0,0 +1,76 @@
+package seedu.address.model.transaction;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class to store the date of transaction.
+ */
+public class Date {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Date should be in the format DD/MM/YYYY";
```

Comment:
Would it be risky/buggy if we use `/`? Since our prefixes uses `/` as well. A bug might occur if the user uses a letter from one of our prefixes by mistake.